src/create-glamorous.js

```diff
@@ -0,0 +1,162 @@
+/*
+ * This is a relatively small abstraction that's ripe for open sourcing.
+ * Documentation is in the README.md
+ */
```

Comment:
Can we actually remove the comment above? This was a comment that I wrote when glamorous was still in the PayPal codebase :smile: